Purchases/FoundationExtensions/OperationQueue+Extensions.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  OperationQueue+Extensions.swift
+//
+//  Created by Joshua Liebowitz on 1/20/22.
+
+import Foundation
+
+extension OperationQueue {
+
+    func addCacheableOperation(_ operation: CacheableNetworkOperation, cacheStatus: CallbackCacheStatus) {
```

Comment:
I don't love this here, but at the moment I'm out of ideas for the best location.